### ISSUE TITLE
Check For Alts On Org Invites

### ISSUE THREAD

**Issue by rozzingit** [Arx-Game/arxcode#31]
**Is your feature request related to a problem? Please describe.**
Sometimes people manage, by accident or otherwise, to get two alts into the same org. Mostly this seems to happen in the Faith org where people have alts in two different discipleships. It's a little awkward to notice and ping staff about. (Although it's been a very long time since this happened that I recognized alts to notice, to be fair. But this request was originally submitted a while ago when it had happened a couple times.)

**Describe the solution you'd like**
It might be worth it to either block people from accepting org invites if their alt is already in the @org (so probably adjusting the @org/accept command) or just sending an automated +job to staff or something letting them know that it happened. I figure with the former option doing it on @org/accept instead of @org/invite will help to maintain a little more alt privacy -- that is, the person being invited would get a note that they can't accept an org invite because they have an alt in that org. Or perhaps it goes in the automated @inform letting them know they have an invite, adding a line that they can't currently accept because their alt is in the org.

**Comment by Packetdancer** [comment]
The code is actually already aware of this situation, and notes it on the staffinfo channel for review when someone joins an org on two alts.  It wouldn't be hard to modify it to just block entirely, if it's still a significant problem.

**Comment by rozzingit** [comment]
Lol. You may have adjusted that when I first submitted the suggestion! I remember you or someone else going "oh yeah that's a good idea." But I still had an open feature request, so it's possible someone just forgot to close it. That seems a good enough solution unless anyone is worried about it happening when staff isn't paying attention to the staffinfo channel. It's not actually a problem that I'm still seeing recurring the way I used to.

**Comment by TehomCD** [comment]
Yeah, probably worth blocking it entirely, even if it doesn't seem like a huge issue.

**Comment by rozzingit** [comment]
Lol. I actually just heard from someone who picked up a new roster where they totally didn't even noticed the PC sharing an org with their main PC. Which isn't a huge deal given that they noticed after, and they put the PC back on the roster anyways for other reasons. So while I don't think it's a huge issue, if you _do_ end up addressing it, that's just kind of another instance of it cropping up.
